src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoDatasetConnectedTest.java

```diff
@@ -156,6 +156,11 @@ public void datasetTest() throws Exception {
           BlobInfo.newBuilder(bucket, targetPath + "ingest-test-sample-no-id.json").build();
       BlobInfo nullPkBlob =
           BlobInfo.newBuilder(bucket, targetPath + "ingest-test-sample-null-id.json").build();
+      BlobInfo nulledNonNullBlob =
```

Comment:
How about using "required" here for consistency/readability?

```suggestion
      BlobInfo nullRequiredBlob =
```